control/preimage/src/commands.rs

```diff
@@ -31,9 +31,18 @@ use crate::bridge_hub_runtime::RuntimeCall as BridgeHubRuntimeCall;
 #[cfg(feature = "polkadot")]
 pub mod asset_hub_polkadot_types {
     pub use crate::asset_hub_runtime::runtime_types::staging_xcm::v4::{
-        junction::Junction::AccountKey20, junction::Junction::GlobalConsensus, junction::NetworkId,
-        junctions::Junctions::X2, location::Location,
+        junction::Junction::AccountKey20,
+        junction::Junction::GlobalConsensus,
+        junction::NetworkId,
+        junctions::Junctions::{X1, X2},
+        location::Location,
     };
+    pub fn get_ether_id(chain_id: u64) -> Location {
+        return Location {
+            parents: 2,
+            interior: X1([GlobalConsensus(NetworkId::Ethereum { chain_id })]),
+        };
+    }
```

Comment:
Please check https://github.com/paritytech/polkadot-sdk/pull/6855#discussion_r1900322845

Reply:
This was merged by parity before discussing with Vincent. But I think we would like it to be `{ parents: 2, interior: X1([GlobalConsensus(Ethereum { chain_id: 1 })]) }` because in XCM this is the convention. I.e. The relaychain token and the destination of the relaychain share the same location. So in our case the bridge location and the bridge native token also shares the same location.